### ISSUE TITLE
Feature request/discussions. Budget pass threshold.

### ISSUE THREAD

**Issue by borris345** [PIVX-Project/PIVX#2730]
Currently the the budget proposals only require a positive +10% vote to pass. Due to the lack of votes in some cases and some people holding large amounts of piv the dao is easily manipulated by a few and is clearly visible to everyone outside and inside of pivx.

I suggest upping the threshold to 30%+ which would create a fairer system where a consensus is needed between more of the voting masternode owners including those who may only have a few, as their vote is then needed.

I'll add in again to unban me from discord. A few (Eric and the crew) have never liked my questioning where funds are spent to get some accountability. Again this points to a problem with the dao as those who have different opinions are silenced.

**Comment by EricStanek** [comment]
Let's be clear; the 10% net yes votes is for the proposal to PASS. That does not necessarily mean it will be funded. If there is limited budget, and other proposals have more net Yes votes, they will get funded and the others will not.

Sounds like what you are looking for, is a number that gives you power, and you won't be happy until the number is high enough that you can kill every proposal like you have tried to in the past. That's not how it works. You need to buy more Masternodes to get more power. I invite you to do so. It would prompt some 'lurking' Masternode Owners to vote more often. Recently, we saw well over 900 votes on a single proposal!

I have never been against you questioning where funds go. My point has always been, you can ask, but proposal owners can simply refuse to tell you. Sometimes that is because they have signed NDAs, and sometimes it is because they know you just want to micro-manage every single little detail. Not cool. PIVX is about Freedom, and it they want to be free of your constant complaints, they can just refuse to cooperate with you. In other words, refusing to cooperate with you is not a 'rule' anyone set. It just is.

If you want to fix this, the solution is simple; just submit a competing proposal! In fact, a healthy proposal system has many proposals that simply can't be funded. That eliminates mediocre proposals. Problem is though, more important than skills offered or price stated in the proposal, is that no one in PIVX trusts you, and THAT is the most important aspect of ANY proposal.

**Comment by EricStanek** [comment]
I should add .... different opinions are always welcome. That is not the issue here. But, your abusive approach and language is not welcome.

If you feel that the DAO should allow for that, then please feel free to start your own PIVX Discord. That would help increase decentralization.

**Comment by borris345** [comment]
How would increasing the vote threshold increase my power or anyone else's? If anything it dilutes everyone's and requires a more of a dao consensus on a passing proposal instead of being able to be pushed through by a few. 

I welcome the unban that will appear then? If they don't want to talk that's fine but a different view is essential and should be welcomed in any dao. Not ban the view that doesn't fit the narrative you try to create. The price speaks for itself and questioning what's going on should be welcomed by all.

Your comments only solidify my argument that the control is to centralised and a different approach is atleast discussed.

**Comment by EricStanek** [comment]
Looking at the current proposals, Marketing Fund proposals are sitting at 29% net yes votes, that's why you want 30%. lol.

What you need to do, is prove that the 10% rule is failing. Which proposals submitted have failed to pass that should have? (Hint: None.) All you have stated so far, is that you can't push it down to 9%, but you can stop it from hitting 30%. 

Sorry. I can't unban you. It was a unanimous group decision of at least 10 admins.

Again, as stated above. You were not banned for your point of view. You were banned for your abusive approach and language. Please stop projecting.

In case you haven't noticed, the entire crypto market just crashed. So, pointing the price is very lame. Besides, the PIVX tech is stellar, and all that lacks is Business Development and Marketing. Yet, you want to kill those proposals. Does not compute. If you don't like the Business Development and Marketing proposals, please vote the current ones down, submit ones you approve of and vote for those. That's how Governance works.

**Comment by borris345** [comment]
I have not even got any masternodes and thus have not been voting. This is discussions to make the voting more fairer and this is the only place i am able to discuss in public with being banned from discord and the forums.  20% more of a reasonable number in your eyes if 30% seems a to high number due to the lack of voting? 10% is a low number and is too easily manipulated. 

It's clearly obvious these numbers threaten your control which is why your so objective.

Who voted these admins in? They were just the ones who started the discord back when dnet started. Then the click got added slowly. People have been pushed out along the way, cryptosi, rhubarb, people thst made pivx what it is especially when they had different views. If everyone sees this as the official discord then a vote should be passed by the community not the discord admins as they are and should not be involved in any decisions. They are nothing but a community member that started it or got placed. The control over all areas has been centralised from the start and this discussion is to try and limit this in the masternode vote.

If the discord is just a community one yes feel free to ban but if others were to be made they also should be added to the main pivx site.

**Comment by EricStanek** [comment]
Again, where is the evidence that the voting is not fair? Which proposals have failed to get funded that you think should have? Sounds like you are suggesting that the existing proposals should fail to get funded, and the minimal Treasury funds we have, don't get used at all. 

So, if you have some logic, or some facts, that would help make your point. But, I'm sure you can appreciate that the devs are not going to make a change to the Voting protocol, because one person that no one trusts, has a 'feeling' things should be different, and doesn't even care enough about voting to have his PIVX in Masternodes so that he can actually vote.

The Discord admins were voted in over time based on their reputations and value offered to PIVX. Yes, people have been pushed out. They did that to themselves. One was abusive towards a Core Developer, and another required more funding than we could justify, and that was not received well.

We have tried over and over and over to explain this to you. PIVX is Decentralized. No one can stop you form buying or selling PIVX. No one can stop you from Staking or running a Masternode. If you have a Masternode, no one can stop you from voting.  

However, that does not mean that the PIVX.org site and the official social channels are open to anyone. If a person trolls, attacks, doxes, or breaks other rules, they will be banned. You, and everyone out there, are welcome to start your own PIVX website, Discord and social channels. That increases decentralization. 

Good luck.

**Comment by borris345** [comment]
I'm not saying that any of the proposals should fail or that some should of been funded that weren't. I'm saying if anyone controls so many masternodes they control to much voting power so uping the threshold for a pass would limit this as a more consensus would be needed across all masternode owners foe anything to pass. Myself and pretty much everyone knows you do and everything is passed through or atleast by you. You're very vocal as you would lose control. I thought you  were a Liberal lefty and were all for distrubuting control and wealth? The dao was made for this reason? To put the power in more than a few? So adding a higher pass threshold would decentralise it even more. More discussions would be needed to create a middle ground for a majority to agree on not just a few.

**Comment by EricStanek** [comment]
Yes. In the beginning, there was too much control with one person. That's pretty much how every project starts out. Ironically, the DAO pushed that one person out, even with all that power. Why? Because the MN Votes ONLY control flow of Treasury funds. The DAO itself is far more than votes. It is Community.

But, as with any project, over time, that vote control is distributed. PIVX is no different. You like to frame it a certain way, but the truth we all see, is that you are not sincere. What you really want, is that YOU have control. It will never happen. Any one whale could move all their PIVX from Masternodes to Staking, and you could fire up your Masternodes again, and you would still fail to get your way. In other words, the opportunity for you or any one person to control the flow of the Treasury funds has passed. This is normal. This is good.

Having said all that, I am not convinced that a higher threshold would work. We often have to battle voter apathy. Or, the opposite could happen. Every single Masternode could vote, yet the proposal still fails because the net Yes was only 29%? Imagine conventional elections working that way. There would be stalemates ALL THE TIME. Countries would never have a leader.

Still looking for facts and logic to support your point.

**Comment by borris345** [comment]
All or you see? This was a discussion on how the voting system could be more decentralized. I gain nothing by making it harder for a single person to be able to block or push through a proposal. It only helps with the decentralisation meaning more masternode owners need to gather around a single proposal agreement. All these discussions with yourself have shown is how your a central entity who would not like more diluted votes.

Good day sir! Hopefully others will chime in.

**Comment by EricStanek** [comment]
Would be great to see what Dash has done. We inherited their rules, and they have grown larger. What have they changed? Anything? Did it help or make things worse? Would be better if we knew the reasoning behind why things were designed the way they were in the first place.
